understanding/21/content-on-hover-or-focus.html

```diff
@@ -1,145 +1,141 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
-	<head>
-		<meta charset="UTF-8"/>
-		<title>Understanding Content on Hover or Focus</title>
-		<link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/>
-	</head>
-	<body>
-		<h1>Understanding Content on Hover or Focus</h1>
 
-		<section id="brief">
-			<h2>In brief</h2>
-			<dl>
-				<dt>Goal</dt><dd>More users can perceive and dismiss non-persistent content.</dd>
-				<dt>What to do</dt><dd>If hover or focus causes content changes, ensure interaction is predictable.</dd>
-				<dt>Why it's important</dt><dd>Unpredictable temporary content can be hard for some to consume and may disrupt others.</dd>
-			</dl>
-		</section>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Understanding Content on Hover or Focus</title>
+    <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove" />
+  </head>
 
-		<section id="intent">
-			<h2>Intent</h2>
-			<p>Additional content that appears and disappears in coordination with keyboard focus or pointer hover often leads to accessibility issues. Reasons for such issues include:</p>
-			<ul>
-				<li>the user may not have intended to trigger the interaction</li>
-				<li>the user may not know new content has appeared</li>
-				<li>the new content may intefere with a user's ability to do a task</li>
-			</ul>
 
-			<p>Examples of such interactions can include custom tooltips, sub-menus and other nonmodal popups which display on hover and focus. The intent of this success criterion is to ensure that authors who cause additional content to appear and disappear in this manner must design the interaction in such a way that users can:</p>
-				<ul>
-					<li>perceive the additional content AND</li>
-					<li>dismiss it without disrupting their page experience.</li>
-				</ul>
+  <body>
+    <h1>Understanding Content on Hover or Focus</h1>
+    <section id="intent">
+      <h2>Intent</h2>
+      <p>Additional content that appears and disappears in coordination with keyboard focus or pointer hover often leads to accessibility issues. Reasons for such issues include:</p>
+      <ol>
```

Comment:
this change to numbered list looks odd ... not sure what other things may have happened here, but that's the one that jumps out at me /cc @kfranqueiro - maybe worth doing a clean re-do of this PR that literally just finds/replaces the words and nothing else (leaving the reformatting for a separate commit or even leaving it as it currently is)